source/eventcore/driver.d

```diff
@@ -322,7 +322,8 @@ interface EventDriverSockets {
 			target_address = Optional default target address. If this is
 				specified and the target address parameter of `send` is
 				left to `null`, it will be used instead.
-
+			options = Optional options for datagram creation. If unset,
+				'DatagramCreateOptions.init' is used.
```

Comment:
Actually, two things: This uses backtips instead of quotes. In DDOC, a word between backtips is equivalent to a word between `$(D )` macro. Also you removed a newline used to separate sections (I don't think this has an incidence on the generated doc, but it separates them better IMO).